lib/replicate.js

```diff
@@ -85,6 +85,15 @@ function Peer (feed, opts) {
   this._destroyed = false
   this._defaultDownloading = this.downloading
   this._iterator = this.remoteBitfield.iterator()
+
+  if (opts.stats) {
+    this._stats = {
```

Comment:
this._stats = opts.stats ? null : { ... }

Reply:
:+1: